From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Wed, 8 Feb 2017 12:05:08 -0500
Subject: common: Avoid warning about implicit declaration of gnupg_fd_valid.
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

* common/logging.c: Add #include "sysutils.h".

--

Without this, we see:

logging.c:573:9: warning: implicit declaration of function \
  ‘gnupg_fd_valid’ [-Wimplicit-function-declaration]
   if (! gnupg_fd_valid (fd))
         ^~~~~~~~~~~~~~

Signed-off-by: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
(cherry picked from commit 8810314e377a9cb6612150a57cf99260ed0bb9f6)
---
 common/logging.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/common/logging.c b/common/logging.c
index ac130535c..670affb12 100644
--- a/common/logging.c
+++ b/common/logging.c
@@ -61,6 +61,7 @@
 #include "i18n.h"
 #include "common-defs.h"
 #include "logging.h"
+#include "sysutils.h"
 
 #ifdef HAVE_W32_SYSTEM
 # define S_IRGRP S_IRUSR
